examples/01_datasets/plot_neurovault_io.py

```diff
@@ -0,0 +1,94 @@
+"""
+
+.. _datasets2:
+
+===========================================
+Using Neurovault Statistical Maps in NiMARE
+===========================================
+
+"""
+###############################################################################
+# Neurovault + NiMARE: Load freely shared statistical maps for Meta-Analysis
+# --------------------------------------------------------------------------
+# `Neurovault <https://neurovault.org/>`_ is an online platform that hosts
+# unthresholded statistical maps, including group statistical maps.
+# NiMARE can read these statistical maps when given a list of collection_ids.
+# I search "working memory" on neurovault, and find these relevant collections:
+#
+# * `2884 <https://neurovault.org/collections/2884/>`_,
+# * `2621 <https://neurovault.org/collections/2621/>`_,
+# * `3085 <https://neurovault.org/collections/3085/>`_,
+# * `5623 <https://neurovault.org/collections/5623/>`_,
+# * `3264 <https://neurovault.org/collections/3264/>`_,
+# * `3192 <https://neurovault.org/collections/3192/>`_,
+# * `457 <https://neurovault.org/collections/457/>`_.
```

Comment:
The example looks great overall, but I don't love how this part renders...

![image](https://user-images.githubusercontent.com/8228902/110143565-5c818500-7da5-11eb-9943-b9b4444dd8e5.png)

Reply:
to be clear, it's the commas/period that's off right? or is there something else as well?

Reply:
I think dropping the commas and period would be good, or keeping them but putting the items on a single line like a sentence. Either way would look good.